collect_app/src/main/java/org/odk/collect/android/formhierarchy/FormHierarchyActivity.java

```diff
@@ -924,4 +927,11 @@ public void deleteGroup() {
             goUpLevel();
         }
     }
+
+    @Override
+    protected void onDestroy() {
+        super.onDestroy();
```

Comment:
I think it is generally recommended to call `super.onDestroy()` after other operations in onDestroy() and that's what we do in other activities.